### PR TITLE
Events

### PR DESCRIPTION
**Pull request by reginafcompton**
Updates for Metro event scraper @fgregg

### CODE REVIEW

File: lametro/events.py [hunk 1]
```diff
@@ -1,32 +1,47 @@
 from legistar.events import LegistarAPIEventScraper
+from legistar.events import LegistarEventsScraper
 
 import requests
+import lxml.html
```
Comment (fgregg) [inline lametro/events.py]: you don't use this import

File: lametro/events.py [hunk 2]
```diff
@@ -1,32 +1,47 @@
...
     WEB_URL = 'https://metro.legistar.com'
+    EVENTSPAGE = "https://metro.legistar.com/Calendar.aspx"
     TIMEZONE = "America/Los_Angeles"
 
     def scrape(self):
+        web_results = self.scrapeWebCalendar()
+
         for event in self.events():
+            # Create a key for lookups in the web_results dict.
+            key = (event['EventBodyName'].strip(), datetime.strptime(event['EventDate'][:10], '%Y-%m-%d').strftime('%-m/%-d/%Y'), event['EventTime'])
```
Comment (fgregg) [inline lametro/events.py]: why are you casting this to a string?

File: lametro/events.py [hunk 3]
```diff
@@ -1,32 +1,47 @@
...
+
+            try:
+                # Look for the event in the web_results dict.
+                web_event_dict = web_results[key]
```
Comment (fgregg) [inline lametro/events.py]: how about 

```python
web_event_dict = web_results.get(key, {'Meeting Details': 'Meeting\xa0details', 'Audio': 'Not\xa0available', 'Recap/Minutes': 'Not\xa0available'})
```

Comment (reginafcompton) [inline lametro/events.py]: Oh, so much better.

File: lametro/events.py (hunk 2 above)
Comment (reginafcompton) [inline lametro/events.py]: I changed this: the output of scrapeWebCalendar returns a datetime.datetime class and so does the scrape function. I'll push those changes.

**Comment by reginafcompton** [comment]
@fgregg New code for your review....

**Comment by reginafcompton** [comment]
@fgregg I made use of the toTime method, but not the toDate method, since the LegistarAPIScraper does not have toDate: https://github.com/opencivicdata/python-legistar-scraper/blob/master/legistar/base.py#L189

Do you prefer this to the hand-made conversions? (I do not like the re-assignment of “self.date_format”).

**Pull request merged**